### PR TITLE
Refactor/check 437 items should belong to one list

### PR DESCRIPTION
**Pull request by melsawy**

### CODE REVIEW

File: db/migrate/20210407060253_add_project_id_to_project_media.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddProjectIdToProjectMedia < ActiveRecord::Migration
+  def change
+  	add_column :project_medias, :project_id, :integer
+  	add_index :project_medias, :project_id
```
Comment (caiosba) [inline db/migrate/20210407060253_add_project_id_to_project_media.rb:4]: Please fix this indentation.

**Review by caiosba — CHANGES_REQUESTED** [review]

File: lib/tasks/data/pg_export.rb [hunk 2]
```diff
@@ -450,7 +443,6 @@ def self.export_team_to_sqlite_lz4_file(team_slug, path)
             TableStrategies::Media,
             TableStrategies::Project,
             TableStrategies::ProjectMedia,
-            TableStrategies::ProjectMediaProject,
```
Comment (caiosba) [inline lib/tasks/data/pg_export.rb:453]: @melsawy please be sure that this rake task is still able to export "project_media_projects" data, even without the model.

Comment (melsawy) [reply lib/tasks/data/pg_export.rb:453]: OK will back this link, How can I run this rake task locally and validate the output?

File: db/migrate/20210407060253_add_project_id_to_project_media.rb (hunk 1 above)
Comment (melsawy) [reply db/migrate/20210407060253_add_project_id_to_project_media.rb:4]: Done

File: lib/tasks/data/pg_export.rb (hunk 2 above)
Comment (caiosba) [reply lib/tasks/data/pg_export.rb]: `rake check:data:export_team`

**Pull request merged**